Lib/test/test_mailcap.py

```diff
@@ -1,10 +1,17 @@
-import mailcap
-import os
 import copy
+import os
+import sys
 import test.support
-from test.support import os_helper
 import unittest
-import sys
+import warnings
+from test.support import os_helper
+
+
+with warnings.catch_warnings():
+    # mailcap is deprecated
+    warnings.simplefilter('ignore', DeprecationWarning)
+    import mailcap
```

Comment:
Like https://github.com/python/cpython/pull/91958:

```suggestion
mailcap = warnings_helper.import_deprecated('mailcap')
```

Reply:
Done.

I looked at asyncore implementation to write this PR, and test_asyncore.py doesn't use this pattern. Maybe you can propose a PR to uniformize the code for all deprecated modules.

Reply:
Thanks! Yeah, I did `test_asyncore.py` and then Brett came up with a neater way for his :) Yep, will have a look.